ui/ducks/app/app.ts

```diff
@@ -82,7 +161,7 @@ export default function reduceApp(state = {}, action) {
       return {
         ...appState,
         alertOpen: true,
-        alertMessage: action.value,
+        alertMessage: action.payload,
```

Comment:
ALERT_OPEN is fired from the showAlert method in actions, which hasn't yet been updated to pass payload, this change should be reverted. We only want to change to use payload on the actions you have specifically typed as PayloadAction in this file. Others will be converted by me in the actions typescript conversion :)

Reply:
Thanks for pointing it out, now i understand the flow here now 🙏